2022-12-12T08:46:04.4457961+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2022-12-12T08:46:04.4628791+07:00  [INF] Hosting environment: "escs_dev" (c3307c92)
2022-12-12T08:46:04.4631798+07:00  [INF] Content root path: "D:\ESCS\ESCS" (b5d60022)
2022-12-12T08:46:04.4969171+07:00 40000127-0002-fd00-b63f-84710c7967bb [WRN] Failed to determine the https port for redirect. (ca76cc21)
2022-12-12T09:56:17.6183223+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2022-12-12T09:56:17.6292177+07:00  [INF] Hosting environment: "escs_dev" (c3307c92)
2022-12-12T09:56:17.6294201+07:00  [INF] Content root path: "D:\ESCS\ESCS" (b5d60022)
2022-12-12T09:56:17.6450811+07:00 40000129-0004-ff00-b63f-84710c7967bb [WRN] Failed to determine the https port for redirect. (ca76cc21)
2022-12-12T09:57:35.7707577+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2022-12-12T09:57:35.7762176+07:00  [INF] Hosting environment: "escs_dev" (c3307c92)
2022-12-12T09:57:35.7763324+07:00  [INF] Content root path: "D:\ESCS\ESCS" (b5d60022)
2022-12-12T09:57:35.7891816+07:00 40000041-0001-fa00-b63f-84710c7967bb [WRN] Failed to determine the https port for redirect. (ca76cc21)
2022-12-12T10:03:49.6070571+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2022-12-12T10:03:49.6117926+07:00  [INF] Hosting environment: "escs_dev" (c3307c92)
2022-12-12T10:03:49.6118781+07:00  [INF] Content root path: "D:\ESCS\ESCS" (b5d60022)
2022-12-12T10:03:49.6199552+07:00 4000004f-0001-fa00-b63f-84710c7967bb [WRN] Failed to determine the https port for redirect. (ca76cc21)
2022-12-12T10:09:34.1052585+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2022-12-12T10:09:34.1125938+07:00  [INF] Hosting environment: "escs_dev" (c3307c92)
2022-12-12T10:09:34.1127050+07:00  [INF] Content root path: "D:\ESCS\ESCS" (b5d60022)
2022-12-12T10:09:34.1262235+07:00 40000158-0004-ff00-b63f-84710c7967bb [WRN] Failed to determine the https port for redirect. (ca76cc21)
2022-12-12T13:23:43.0870962+07:00 40000185-0000-fb00-b63f-84710c7967bb [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.NullReferenceException: Object reference not set to an instance of an object.
   at ESCS.COMMON.ExtensionMethods.RequestExtensionMethod.GetDataRequestNew(HttpRequest rq, escs_nguoi_dung nguoi_dung) in D:\ESCS\ESCS.COMMON\ExtensionMethods\RequestExtensionMethod.cs:line 250
   at ESCS.Areas.Admin.Controllers.CarConfigurationController.GetDsNgay() in D:\ESCS\ESCS\Areas\Admin\Controllers\CarConfigurationController.cs:line 110
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at ESCS.Startup.<>c.<<Configure>b__5_0>d.MoveNext() in D:\ESCS\ESCS\Startup.cs:line 127
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Session.SessionMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Session.SessionMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.StatusCodePagesMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Server.IIS.Core.IISHttpContextOfT`1.ProcessRequestAsync()
2022-12-12T13:23:43.0881284+07:00 40000185-0000-fb00-b63f-84710c7967bb [WRN] The response has already started, the error page middleware will not be executed. (e1dd4c0d)
2022-12-12T13:23:43.1071611+07:00 40000185-0000-fb00-b63f-84710c7967bb [ERR] Connection ID ""18086456104325218692"", Request ID ""40000185-0000-fb00-b63f-84710c7967bb"": An unhandled exception was thrown by the application. (bf316047)
System.NullReferenceException: Object reference not set to an instance of an object.
   at ESCS.COMMON.ExtensionMethods.RequestExtensionMethod.GetDataRequestNew(HttpRequest rq, escs_nguoi_dung nguoi_dung) in D:\ESCS\ESCS.COMMON\ExtensionMethods\RequestExtensionMethod.cs:line 250
   at ESCS.Areas.Admin.Controllers.CarConfigurationController.GetDsNgay() in D:\ESCS\ESCS\Areas\Admin\Controllers\CarConfigurationController.cs:line 110
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at ESCS.Startup.<>c.<<Configure>b__5_0>d.MoveNext() in D:\ESCS\ESCS\Startup.cs:line 127
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Session.SessionMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Session.SessionMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.StatusCodePagesMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Server.IIS.Core.IISHttpContextOfT`1.ProcessRequestAsync()
2022-12-12T13:28:55.2782090+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2022-12-12T13:28:55.2855709+07:00  [INF] Hosting environment: "escs_dev" (c3307c92)
2022-12-12T13:28:55.2856837+07:00  [INF] Content root path: "D:\ESCS\ESCS" (b5d60022)
2022-12-12T13:28:55.2992811+07:00 40000247-0001-fc00-b63f-84710c7967bb [WRN] Failed to determine the https port for redirect. (ca76cc21)
